submissions/benjaminKamena/PHP/numbers.php

```diff
@@ -0,0 +1,13 @@
+<?php
+//PHP program that displays every number between 100 and 150 each on a new line using a loop
+for($x = 100; $x <= 150; $x++){
```

Comment:
This output will display both `100` and `150`. But the questions say `displays every number between 100 and 150`. The keyword there is `between`